packages/builder-util-runtime/src/httpExecutor.ts

```diff
@@ -446,6 +446,7 @@ function configurePipes(options: DownloadCallOptions, response: IncomingMessage)
   let lastStream = response
   for (const stream of streams) {
     stream.on("error", (error: Error) => {
+      fileOut.close()
```

Comment:
Why is no callback being supplied here? (as it is at line 457)
```
;(fileOut.close as any)(options.callback)
```
Shouldn't we be doing so?

Something like:
```
const closeFile = () => { ;(fileOut.close as any)(options.callback) }
```
Then use `closeFile()` both here and at L457.